bigtable/quickstart/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google LLC
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START bigtable_quickstart]
+// Quickstart is a sample program demonstrating use of the Cloud Bigtable client
+// library to read a row from an existing table.
+package main
+
+import (
+	"flag"
+	"log"
+	"context"
+	"cloud.google.com/go/bigtable"
+)
+
+// User-provided constants.
+const (
+	rowKey = "r1"
+	columnFamilyName = "cf1"
+)
+
+func main() {
+	projectID := flag.String("project", "", "The Google Cloud Platform project ID. Required.")
+	instanceID := flag.String("instance", "", "The Google Cloud Bigtable instance ID. Required.")
+	tableID := flag.String("table", "", "The Google Cloud Bigtable table ID. Required.")
+	flag.Parse()
```

Comment:
I don't *think* any of our quickstarts use flags. I think the style is to have these as constants.

Reply:
Can you take a look at the update: left flag in as an override option(will not be visible in the docs). I was trying to keep it easy to test as well.

Reply:
I like this - looks good in the docs and is easy to use from command line.